Modules/Relationships/src/Relationships.ConsumerApi/Controllers/RelationshipsController.cs

```diff
@@ -139,9 +126,6 @@ public async Task<IActionResult> TerminateRelationship([FromRoute] string id, Ca
     [ProducesError(StatusCodes.Status404NotFound)]
     public async Task<IActionResult> RelationshipReactivationRequest([FromRoute] string id, CancellationToken cancellationToken)
     {
-        var peerOfActiveIdentityInRelationshipResponse = await _mediator.Send(new GetPeerOfActiveIdentityInRelationshipQuery { Id = id }, cancellationToken);
-        await EnsurePeerIsNotToBeDeleted(peerOfActiveIdentityInRelationshipResponse.IdentityAddress, cancellationToken);
-
```

Comment:
I think this also doesn't make much sense, i.e. to send relationship reactivation request to the identity that is to be deleted. Just letting @tnotheis know.

Reply:
Waiting on Timo's answer.